src/grid.py

```diff
@@ -713,5 +713,5 @@ def reinitialization(self,initialization,microbes_pp,output,mode,pulse,*args):
         # last: assign microbes to each grid box randomly based on prior densities
         choose_taxa = np.zeros((self.n_taxa,self.gridsize), dtype='int8')
         for i in range(self.n_taxa):
-            choose_taxa[i,:] = np.random.choice([1,0], self.gridsize, replace=True, p=[frequencies[i], 1-frequencies[i]])
+            choose_taxa[i,:] = np.random.binomial(1, frequencies[i], self.gridsize)
```

Comment:
When I changed the numbers back to "float32" this sampling started failing saying that the probabilities do not sum to 1.0. I presume they must be converted to "float64" before getting summed inside `np.random.random_choice`. 

I have changed the sampling to `binominal`, which i believe should be equivalent and would avoid the summation errors.